db/migrate/20171124131334_create_certificates.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCertificates < ActiveRecord::Migration
+  def change
+    create_table :certificates do |t|
+      t.text :certificate
+      t.date :expiration
+
+      t.timestamps null: false
```

Comment:
Minor: `null: false` is not required here as activerecord will automatically fill `created_at` and `updated_at`.